.gitattributes

```diff
@@ -0,0 +1,62 @@
+# Set default behavior to automatically normalize line endings.
```

Comment:
Do we really need this? I don't remember where we are towards line endings. @RyanSkraba maybe?